website/generator/__main__.py

```diff
@@ -42,17 +42,24 @@ def main():
 
     This function is invoked automatically by mkdocs during the build process.
     """
-    for entry in website.generator.database.cv:
+
+    database = website.generator.database.cv
+    # Create a single page for each entry in the database
+    for entry in database:
         with mkdocs_gen_files.open(
             os.path.join("cv", "entries", f"{entry.identifier}.md"), "w"
         ) as markdown:
             markdown.write(
                 render(
                     "pages/cv_entry.md",
-                    database=website.generator.database.cv,
+                    database=database,
                     entry=entry,
                 )
             )
+    # Create an overview page with tabulated and linked entries.
+    with mkdocs_gen_files.open(os.path.join("cv", "index.md"), "w") as markdown:
```

Comment:
Why do we want to concatenate two files here? It's easier to just render `cv_overview_table.md` inside `cv.md`.